libs/go/zmscli/cli.go

```diff
@@ -23,7 +24,9 @@ import (
 const (
 	// JSONOutputFormat is the JSON output format for commands.
 	JSONOutputFormat = "json"
-	// DefaultOutputFormat is the default YAML output format for commands.
+	// YAMLOutputFormat is the YAML output format for commands.
+	YAMLOutputFormat = "yamlNew"
```

Comment:
For now, to show the "generic Yaml" format we'll use the yamlNew as the output flag. Once we get rid of the "manual yaml" we'll replace this flag with a simple "yaml".

Reply:
we should call the format as yaml and then update zms-cli to default to manualYaml (the old format). This way it allows users to start using -o yaml right now with the new format and not change later once it becomes the default option

Reply:
Fixed